Xwt.XamMac/Xwt.Mac/ListViewBackend.cs

```diff
@@ -140,14 +152,36 @@ void HandleDoubleClick (object sender, EventArgs e)
 				});
 			}
 		}
-		
+
+		public override NSTableColumn AddColumn (ListViewColumn col)
+		{
+			NSTableColumn tcol = base.AddColumn (col);
+			var widths = new List<nfloat> ();
+			if (Table.RowCount > 0)
+				widths.InsertRange (0, Enumerable.Repeat<nfloat> (-1f, (int)Table.RowCount));
```

Comment:
This will do Insert N times instead of doing a block insert. Avoid direct IEnumerables here. Please pass something `ICollection` derived. (the `IEnumerable<T>` overload handles checking for `ICollection<T>`)

Reply:
Actually, I think pre-allocating list capacity `(new List<nfloat> (table.RowCount)` then doing
`for (int i = 0; i < Table.RowCount; ++i) widths.Add (-1f)` is better